caiman/base/movies.py

```diff
@@ -1047,7 +1058,17 @@ def animate(i):
 
         looping = True
         terminated = False
-
+        if save_movie:
+            #fourcc = cv2.VideoWriter_fourcc('8', 'B', 'P', 'S')
```

Comment:
Why are we checking in commented-out code?

Reply:
@pgunn There is a bunch of different codecs for saving a video through openCV. Each will work in some platforms, but not in others. It would actually be pretty helpful if you could spend some time investigating this, and come up with some rules about which codec to use when.